docs/voxa-event.rst

```diff
@@ -3,15 +3,35 @@
 The ``voxaEvent`` Object
 ===========================
 
-.. js:class:: IVoxaEvent(event, lambdaContext)
+.. js:class:: VoxaEvent(event, context)
 
   The ``voxaEvent`` object contains all the information from the Voxa event, it's an object kept for the entire lifecycle of the state machine transitions and as such is a perfect place for middleware to put information that should be available on every request.
 
-  .. js:attribute:: VoxaEvent.model
+  .. js:attribute:: rawEvent
+
+    A plain javascript copy of the request object as received from the platform
+
+  .. js:attribute:: executionContext
+
+    On AWS Lambda this object contains the context
+
+  .. js:attribute:: t
+
+    The current translation function from i18next, initialized to the language of the current request
+
+  .. js:attribute:: renderer
+
+    The renderer object used in the current request
+
+  .. js:attribute:: platform
+
+    A string attribute with the name of platform for the current request, it can be alexa or dialogflow
```

Comment:
or microsoft/cortana ?

Should we add support in docs for Cortana in the docs ?